js/main.js

```diff
@@ -0,0 +1,40 @@
+function Register() {
+  this.domElement = null;
+  this.total = 0.00;
+
+  this.initialize = function(cont) {
+    this.domElement = $('<div>');
+    cont.append(this.domElement);
+  }
+
+  this.render = function() {
+    var templateString = '<div class="ticket"><h1>Thank You!</h1><table><tbody class="entries"></tbody><tfoot><tr><th>Total</th><th class="total">$0.00</th></tr></tfoot></table></div><form class="entry"><input class="newEntry" autofocus placeholder="How Much?"></form>'
```

Comment:
There isn't a perfect method to deal with adding big blocks of html as strings, which is why templating exists. Having code like this is not very maintainable though and probably would be better served by spliting it up into multiple lines with string concatenation ('something' +
'somethingElse)